vtex/faststore
# Feat: Next 13 - Global Section - Footer Section

## What's the purpose of this pull request?

Migration of the `Footer` section from the pages router to the app router (Next 13).

## How it works?
<img width="720" alt="Screenshot 2024-06-11 at 16 16 02" src="https://github.com/vtex/faststore/assets/19983991/fefd2862-ecfa-4881-8070-d4b13a353216">

## How to test it?

- Run `yarn dev`
- Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
- Make sure `/fs-next-update` is rendering with the `Footer` section on the bottom of the page and without errors.

### Starters Deploy Preview





## References

- [Jira task](https://vtex-dev.atlassian.net/browse/SFS-1020)
